packages/smooth_app/lib/pages/scan/scan_page.dart

```diff
@@ -6,17 +6,14 @@ import 'package:smooth_app/database/product_query.dart';
 import 'package:smooth_app/pages/scan/continuous_scan_page.dart';
 
 class ScanPage extends StatefulWidget {
-  const ScanPage({required this.offstage, required this.navigatorKey});
-
-  final bool offstage;
-  final GlobalKey<NavigatorState> navigatorKey;
+  const ScanPage();
 
   @override
-  State<ScanPage> createState() => _ScanPageState();
+  State<ScanPage> createState() => ScanPageState();
 }
 
-class _ScanPageState extends State<ScanPage> {
-  ContinuousScanModel? _model;
+class ScanPageState extends State<ScanPage> {
+  static ContinuousScanModel? continuousScanModel;
```

Comment:
I'm not convinced by the de-privatization of `_ScanPageState`.
I'm not convinced either by the user of `static` fields (that comes from my Android/java background).